frontend/doc-site/09_Frequently-Asked-Questions.mdx

```diff
@@ -2,4 +2,59 @@
 
 We’ve compiled a list of the questions we get most frequently. Check them out below:
 
-**MAX**
+
+## Cellxgene Data Portal
+
+### How can I find a dataset of interest?
+
+Refer to this [tutorial (TODO)](www.google.com) to find out how the dataportal is organized and how you can search by dataset metadata.
+
+### How can I request a particular dataset to be on the portal?
+
+You can request a particular dataset by [filing a github issue](https://github.com/chanzuckerberg/single-cell-curation/issues/new) on the cellxgene single-cell-curation repo.
+
+### How can I contribute a dataset to the cellxgene data portal?
+
+We are continuing to expand our data corpus and would love to chat further. Reach out to our curation team at [cellxgene@chanzuckerberg.com ](cellxgene@chanzuckerberg.com )
+
```

Comment:
Removing this section based on [slack discussion](https://czi-sci.slack.com/archives/C037WS42XV3/p1653594111210409): 

```suggestion
```